rubocop.yml

```diff
@@ -74,13 +74,6 @@ Style/BlockDelimiters:
   Exclude:
     - 'spec/**/*'
 
-# Prevents noise from Abilities::SomeClass complaints
-# https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_style.md#styleclassandmodulechildren
-Style/ClassAndModuleChildren:
-  EnforcedStyle: compact
```

Comment:
I actually prefer this style.  There are a number of graphql implementations that use the compact syntax in caasbox.

Would it be acceptable to just disable the cop all together?

Reply:
We will have to make a lot of changes, even in other apps to go from compact to nested.

There are actual programmatic reasons to not use compact. Again I prefer compact syntactically/visually.

Example issues:
- https://github.com/Interfolio/intf_platform/pull/5/files#diff-c94f57ac3d9101ec54a7e5d8b03eba02R12 (wouldn't be able to do this with compact)
- https://stackoverflow.com/a/24393133/998322 (constants scoped differently and also other init differences)